aws/table_aws_backup_framework.go

```diff
@@ -158,6 +165,13 @@ func listAwsBackupFrameworks(ctx context.Context, d *plugin.QueryData, _ *plugin
 //// HYDRATE FUNCTIONS
 
 func getAwsBackupFramework(ctx context.Context, d *plugin.QueryData, h *plugin.HydrateData) (interface{}, error) {
+	// AWS Backup audit manager is not supported in all regions
+	// https://aws.amazon.com/about-aws/whats-new/2022/05/aws-backup-audit-manager-adds-amazon-s3-storage-gateway/#:~:text=AWS%20Backup%20Audit%20Manager%20is,Middle%20East%20(Bahrain)%20Regions.
+	region := d.KeyColumnQualString(matrixKeyRegion)
+	if region == "ap-northeast-3" {
```

Comment:
Put a comment as to why we are directly checking the region in code. As opposed to why we are checking with the service in other parts of the code.

Reply:
I have updated the comment; let me know if this looks good.